### PR TITLE
[SDCI] Clarify running pipelines limitations

### PR DESCRIPTION
**Pull request by rodrigo-roca**
### What does this PR do? What is the motivation?
Pipelines with status `running` also encapsulate queued and waiting pipelines. We do not have statuses for those yet so this PR marks this as a known limitation.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by drodriguezhdez — APPROVED** [review]

File: content/en/continuous_integration/pipelines/awscodepipeline.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Set up tracing on AWS CodePipeline to collect data about pipeline executions, an
 | Pipeline Visibility | Platform | Definition |
 |---|---|---|
 | [Partial retries][14] | Partial pipelines | View partially retried pipeline executions. |
-| *[Running pipelines][15] | Running pipelines | View pipeline executions that are running. |
+| *[Running pipelines][15] | Running pipelines | View pipeline executions that are running. Queued or waiting pipelines will show with status "Running" on Datadog. |
```
Comment (brett0000FF) [inline content/en/continuous_integration/pipelines/awscodepipeline.md:32]:
```suggestion
| *[Running pipelines][15] | Running pipelines | View pipeline executions that are running. Queued or waiting pipelines show with status "Running" on Datadog. |
```

File: content/en/continuous_integration/pipelines/github.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Set up tracing in GitHub Actions to track the execution of your workflows, ident
 
 | Pipeline Visibility | Platform | Definition |
 |---|---|---|
-| [Running pipelines][12] | Running pipelines | View pipeline executions that are running. |
+| [Running pipelines][12] | Running pipelines | View pipeline executions that are running. Queued or waiting pipelines will show with status "Running" on Datadog. |
```
Comment (brett0000FF) [inline content/en/continuous_integration/pipelines/github.md:34]:
```suggestion
| [Running pipelines][12] | Running pipelines | View pipeline executions that are running. Queued or waiting pipelines show with status "Running" on Datadog. |
```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ Set up tracing in GitLab to collect data on your pipeline executions, analyze pe
 
 | Pipeline Visibility | Platform | Definition |
 |---|---|---|
+| [Running pipelines][25] | Running pipelines | View pipeline executions that are running. Queued or waiting pipelines will show with status "Running" on Datadog. |
```
Comment (brett0000FF) [inline content/en/continuous_integration/pipelines/gitlab.md:31]:
```suggestion
| [Running pipelines][25] | Running pipelines | View pipeline executions that are running. Queued or waiting pipelines show with status "Running" on Datadog. |
```

**Review by brett0000FF — APPROVED** [review]
Thanks for the update!

**Pull request merged**